perf/NodeJS/Jering.Javascript.NodeJS.Performance.csproj

```diff
@@ -17,7 +17,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <Content Include=".\dummyModule.js" copyToOutputDirectory="PreserveNewest" />
+        <Content Include=".\dummyModule.js" copyToOutputDirectory="Always" />
```

Comment:
Any reason for this?

Reply:
No, I tried to run the benchmark in out of process mode :), rolled back changes